CHANGES.md

```diff
@@ -1,6 +1,7 @@
 # ChangeLog
 
 ## vX.X.X
+* Use CMake variable names from find_package(Python) which are available with CMake 3.12+. A warning is sent when the python version found is 2. This can be changed to FATAL_ERROR at a later stage.
```

Comment:
let's make this a bit clearer. If they passed `PYTHON_EXECUTABLE` before, it'll be ignored. 

Unless we put in some lines
```cmake
if (PYTHON_EXECUTABLE and not Python_EXECUTABLE)
  set(Python_EXECUTABLE %{PYTHON_EXECUTABLE})
endif()
# same for other python variables
```
that might be a good idea anyway?